config/application.rb

```diff
@@ -82,5 +82,6 @@ class Application < Rails::Application
     config.bitbucket_secret = ENV["BITBUCKET_SECRET"]
     config.ga_analytics_id = ENV["GA_ANALYTICS_ID"]
     config.gtm_id = ENV["GTM_ID"]
+    config.tidelift_api_key = ENV["TIDELIFT_API_KEY"]
```

Comment:
I'm setting to a new org key called "Libraries Lifting API Key" in "tidelift-internal" (which has beta API access)